crates/apub/src/activities/create_or_update/comment.rs

```diff
@@ -191,6 +195,18 @@ impl ActivityHandler for CreateOrUpdateNote {
     };
     CommentLike::like(context.pool(), &like_form).await?;
 
+    // Calculate initial hot_rank
+    let conn = &mut get_conn(context.pool()).await?;
+
+    diesel::update(comment_aggregates::table)
```

Comment:
This looks correct, but don't run diesel directly here. Add a helper function to either `crates/db_schema/src/impls/comment.rs`, (or create a new one at `crates/db_schema/src/impls/comment_aggregates.rs`, and call it like:

`CommentAggregates::update_hot_ranks(context.pool(), comment_id).await?`

Reply:
It looks like there's already comment_aggregates logic in `crates/db_schema/src/aggregates/comment_aggregates.rs` - is there any reason to not add this `update_hot_ranks` function there?